docs/source/getting_started/index.md

```diff
@@ -227,6 +227,17 @@ client.tool_runtime.rag_tool.insert(
     documents=documents,
     vector_db_id=vector_db_id,
     chunk_size_in_tokens=512,
+vector_providers = [
```

Comment:
indent needed?

Reply:
I'm sorry, I thought you tested it.
I just updated it with fixes that works